app/assets/stylesheets/ama_layout/layout_components/sections.scss

```diff
@@ -65,6 +73,16 @@
   background: $brand-green;
 }
 
+.section-heading--orange{
```

Comment:
I'm not sure why the modifiers on the section heading are just floating randomly in this file, but they should be nested under the original block, and then &--orange, &--yellow, etc.